cli/azd/pkg/project/framework_service_docker.go

```diff
@@ -19,10 +19,11 @@ import (
 )
 
 type DockerProjectOptions struct {
-	Path     string           `json:"path"`
-	Context  string           `json:"context"`
-	Platform string           `json:"platform"`
-	Tag      ExpandableString `json:"tag"`
+	Path      string           `json:"path"`
+	Context   string           `json:"context"`
+	Platform  string           `json:"platform"`
+	Tag       ExpandableString `json:"tag"`
+	BuildArgs string           `json:"buildArgs"`
```

Comment:
Lets add support for 1 or many build args.  Based on my research it looks like each docker build arg should be set as a separate argument.

Example
```
docker build \
-t essearch/ess-elasticsearch:1.7.6 \
--build-arg number_of_shards=5 \
--build-arg number_of_replicas=2 \
--no-cache .
```

In this case the `BuildArgs` property here should be updated to a `[]string` and then we can do the right thing when calling the docker CLI.